imap_processing/cdfutils/global_base.py

```diff
@@ -0,0 +1,353 @@
+from dataclasses import dataclass
+
+import numpy as np
+
+from imap_processing.cdfutils.defaults import GlobalConstants
+
+
+class Epoch:
+    """Shared attributes for the Epoch coordinate variables."""
+
+    @staticmethod
+    def output():
+        """
+        Return a dictionary with global base attributes.
+
+        Returns
+        -------
+         Dictionary of epoch attributes
+        """
+        return {
+            "CATDESC": "Default time",
+            "FIELDNAM": "Epoch",
+            "FILLVAL": GlobalConstants.INT_FILLVAL,
+            "FORMAT": "a2",
+            "LABLAXIS": "Epoch",
+            "UNITS": "ns",
+            "VALIDMIN": GlobalConstants.MIN_EPOCH,
+            "VALIDMAX": GlobalConstants.MAX_EPOCH,
+            "VAR_TYPE": "support_data",
+            "SCALETYP": "linear",
+            "MONOTON": "INCREASE",
+            "TIME_BASE": "J2000",
+            "TIME_SCALE": "Terrestrial Time",
+            "REFERENCE_POSITION": "Rotating Earth Geoid",
+        }
+
+
+class GlobalBase:
```

Comment:
Since you asked for opinions on naming, I'll give a quick suggestion. Add attrs, or something about cdf to this?

`GlobalAttrs`, would make it explicit that these are CDF Attributes when used elsewhere and inherited. In the IDEX namespace you are using `InstrumentBase` and things, but it is a bit unclear what is in the InstrumentBase. Another option would be to leave the base as well, `InstrumentAttrsBase`, but then it seems like it is getting long.

Overall, I think Base is fine, but just figured I throw out the Attrs to correspond to what we are using these objects for.